src/site/content/en/blog/speed-at-scale/index.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Speed at Scale - What's new in Web Performance? 
+subhead: Learn about three new Web Performance launches from I/O 2019
+authors:
+  - katiehempenius
+  - addyosmani
+date: 2019-05-24
+hero: scale-header.jpg
+alt: Speedometer on a retro car
+description: |
+  For Google I/O 2019, we introduced three new Web Performance initiatives that we hope will lead to better user experiences for everyone.
+tags:
+  - post # post is a required tag for the article to show up in the blog.
+  - performance
+---
+
+
+During the "Speed at Scale" [talk](https://www.youtube.com/watch?v=YJGCZCaIZkQ&feature=youtu.be) at Google I/O 2019 we announced three things that we hope will improve web performance over the coming year.
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/YJGCZCaIZkQ" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
```

Comment:
Good catch. I'll see if playing around with the width/height would still work for the responsive layout for this post, but can otherwise defer to infra if I can't do better :)